src/hashtagPlugin/index.js

```diff
@@ -1,23 +1,33 @@
 import Hashtag from './Hashtag';
 import hashtagStrategy from './hashtagStrategy';
 import { fromJS } from 'immutable';
-import styles from './Hashtag/styles.css';
+import styles from './styles.css';
+import decorateComponentWithProps from '../utils/decorateComponentWithProps';
 
-const defaultHashTagTheme = fromJS({
-  hashtag: styles.root,
+const defaultTheme = fromJS({
+  hashtag: styles.hashtag,
 });
 
-const hashtagPlugin = (config = {}) => ({
-  pluginProps: {
-    decorators: [
-      {
-        strategy: hashtagStrategy,
-        component: Hashtag,
-        theme: defaultHashTagTheme.merge(config.theme),
```

Comment:
I reverted this back in order to keep the API as similar as the original draftjs API: https://facebook.github.io/draft-js/docs/advanced-topics-decorators.html#compositedecorator

In addition this makes it a bit more flexible, because then people who build a plugin just for themselves do not need to know anything about theme.